package.json

```diff
@@ -31,20 +31,20 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
-    "babel-eslint": "^8.0.0",
+    "babel-eslint": "^8.0.2",
     "babel-loader": "^7.1.0",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-webpack-aliases": "^1.0.3",
-    "babel-preset-env": "^1.6.0",
+    "babel-plugin-webpack-aliases": "^1.1.3",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-flow": "^6.23.0",
     "chai": "^4.1.1",
     "cross-env": "^5.1.1",
-    "eslint": "^4.5.0",
+    "eslint": "^4.11.0",
     "eslint-loader": "^1.9.0",
     "eslint-plugin-flowtype": "^2.35.0",
-    "flow-bin": "^0.57.3",
+    "flow-bin": "^0.59.0",
     "mocha": "^4.0.1",
-    "webpack": "^3.5.5"
+    "webpack": "^3.8.1"
```

Comment:
seems all those updates are not linked to your commit, why not create a specific commit update?